src/api/app/controllers/webui/request_controller.rb

```diff
@@ -177,6 +177,26 @@ def request_action
     end
   end
 
+  def request_action_changes
```

Comment:
With the use of this new action, the current action `request_action` could be deleted once the `request_show_redesign` feature is finished. Could you add a comment above `def request_action` for this?

Reply:
Yes. Done.